src/app/components/case-studies/case-detail/case-detail.component.scss

```diff
@@ -174,12 +178,12 @@
 
   blockquote {
     width: 90%;
-    padding: 5px 10px;
+    padding: 24px 28px;
```

Comment:
Are these universal to how we're handling Butter styilings? If so they should probably be in a shared butter-styling scss file or in the sass>styles.scss style sheet.